lms/static/scripts/frontend_apps/components/GroupConfigSelector.js

```diff
@@ -118,18 +134,27 @@ export default function GroupConfigSelector({
         </LabeledCheckbox>
       </div>
       {useGroupSet && (
-        <div className="GroupConfigSelector__select">
-          {fetchError && (
+        <div>
+          {fetchError && isAuthorizationError(fetchError) && (
```

Comment:
The conditionals in the JSX for this component are a little inelegant (if valid). I'd like to clean up this component's markup and structure very soon as part of the tailwind conversion of the `FilePickerApp` component, to which it is currently coupled. That is the only LMS component that hadn't been converted as it requires a little more thought than the other components.